api/config/config.example.json

```diff
@@ -43,6 +43,8 @@
     "AccountID": "XXXXXXXXXX",
     "QueueName": "XXXXXXXXXX"
   },
+  "RFID_BCRYPT_SALT" : "NOT_SET",
```

Comment:
the salt we use to hash passwords is `secretKey` on line 49,, can we just use the same salt?